autoPyTorch/pipeline/base_pipeline.py

```diff
@@ -366,3 +366,19 @@ def get_additional_run_info(self) -> Dict:
             Dict: Additional information about the pipeline
         """
         return self._additional_run_info
+
+    @staticmethod
+    def get_default_pipeline_options() -> Dict[str, Any]:
+        return {
+            'job_id': '1',
+            'working_dir': '/tmp/experiment_1',
+            'device': 'cpu',
+            'budget_type': 'epochs',
+            'epochs': 20,
+            'runtime': 3600,
+            'torch_num_threads': 1,
+            'early_stopping': 10,
+            'use_tensorboard_logger': True,
+            'use_pynisher': False,
+            'metrics_during_training': True
```

Comment:
For my information: what is `metrics_during_training` used for?

Reply:
So, if it is true, then after every epoch the metrics are computed and stored in the summary along with the loss, if it is fale, then only the loss is stored